src/Routes.js

```diff
@@ -56,7 +56,7 @@ const Routes = (props) => (
     <Route path="/home_dac_info" component={HomeDacInfo} />
     <Route path="/home_researcher_info" component={HomeResearcherInfo} />
     <Route path="/election404" component={Election404} />
-    <Route path="/NHGRIpilotinfo" component={NHGRIpilotinfo} />
+    <Route path="/nih_pilot_info" render={(routeProps) => <NIHPilotInfo {...routeProps} {...props} />} />
```

Comment:
@JVThomas' feedback on my PR is correct, this can be simplified
```suggestion
    <Route path="/nih_pilot_info" render={(routeProps) => <NIHPilotInfo />} />
```
It might even be more simplifiable to 
```suggestion
    <Route path="/nih_pilot_info" render={() => <NIHPilotInfo />} />
```